pkg/controllers/provisioning/provisioner.go

```diff
@@ -288,7 +295,9 @@ func (p *Provisioner) NewScheduler(ctx context.Context, pods []*v1.Pod, stateNod
 			}
 		}
 	}
-
+	if len(notReadyNodePools) > 0 {
+		log.FromContext(ctx).WithValues("nodePools", nodePoolList).Info("skipped nodePool(s) due to not ready")
```

Comment:
nit on this, up to you. Do whatever you think looks best. Personally don't think you need the (s)
```suggestion
		log.FromContext(ctx).WithValues("nodePools", nodePoolList).Info("skipped nodePools, not ready")
```